matcher/service/openreview_interface.py

```diff
@@ -100,11 +109,34 @@ def maximums(self):
 
         return self._maximums
 
+    @property
+    def custom_demand_edges(self):
+        if self._custom_demand_edges is None and self.config_note.content.get('custom_user_demand_invitation'):
+            self._custom_demand_edges = self.client.get_grouped_edges(
+                invitation=self.config_note.content['custom_user_demand_invitation'],
+                tail=self.config_note.content['match_group'],
+                select='weight')
+        return self._custom_demand_edges
+
+    @property
+    def custom_supply_edges(self):
+        if self._custom_supply_edges is None and self.config_note.content.get('custom_max_papers_invitation'):
+            self._custom_supply_edges = self.client.get_grouped_edges(
+                invitation=self.config_note.content['custom_max_papers_invitation'],
+                head=self.config_note.content['match_group'],
+                select='tail,weight')
+        return self._custom_supply_edges
+
     @property
     def demands(self):
         if self._demands is None:
-            self._demands = [int(self.config_note.content['max_users']) for paper in self.papers]
```

Comment:
you need to keep this line, `demands` should always be initialized with `max_users` and then override the values if there is an edge from `custom_user_demand_invitation` . This is the same logic we have for custom_loads.

Reply:
Done.